src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -59,7 +59,7 @@ export default class SaveInProgressIntro extends React.Component {
     if (profile.loading && !this.props.resumeOnly) {
       return (
         <div>
-          <LoadingIndicator message="We’re checking to see if you have a saved version of this application"/>
+          <LoadingIndicator message="We’re checking to see if you have a saved version of this application."/>
```

Comment:
Also two more of these in /js/common/schemaform/FormApp.jsx lines 107 and 109

Maybe:

`Retrieving your saved form...`

and

`Saving your form...`

but up to you if those are in scope